debug/watcher.py

```diff
@@ -82,7 +82,12 @@ def __init__(self, dic, line_adapter = None, verbose = False):
                     # No position specification was found.
                     # It's not a breakpoint handler.
                     continue
-                raw_file_name, line = line_adapter.failback()
+                try:
+                    raw_file_name, line = line_adapter.failback()
+                except Exception as e:
+                    raise RuntimeError("Failed to set breakpoint %s\n%s" % (
```

Comment:
May be: Failed to set breakpoint for 'on_board_init_start'

Reply:
Or: Failed to set breakpoint with handler '%s'

Or: keep things shorter and simpler

Reply:
`with handler` is not shorter then `for`

Reply:
Yes, so let it be as it is.